tasks/aiida-deps-debian.yml

```diff
@@ -1,10 +1,15 @@
 ---
+- name: Update apt cache
+  become: true
+  become_user: "{{ root_user }}"
+  apt:
+    update_cache: true
```

Comment:
does one need to set a time to live so that this does not cause changes?

Reply:
That I do not know. Not very familiar with apt. What should I do?